src/MirOS.xml

```diff
@@ -0,0 +1,92 @@
+<SPDX name="MirOS Licence" identifier="MirOS" osi-approved="true">
```

Comment:
Suggest to change spelling of "Licence" to "License" in SPDX name for consistency with other license list names.